requirements/requirements.txt

```diff
@@ -31,6 +30,4 @@ adapt-parser>=0.5.1
 padatious~=0.4.8
 fann2==1.0.7
 padaos~=0.1.9
-precise-runner~=0.2.1
-petact~=0.1.2
```

Comment:
Any idea what this was used for? Anywhere we might need to add it to dependencies?

Reply:
this was from the precise engine, these are defined in our plugin now and no longer part of core